src/ua-parser.js

```diff
@@ -452,14 +459,6 @@
       [MODEL, [VENDOR, SAMSUNG], [TYPE, MOBILE]],
       [
         // Apple
-        /((ipod|iphone)\d+,\d+)/i // iPod/iPhone model
```

Comment:
Thanks for contributing to amplitude. Can you comment out this code? 
This is what we added in order to read the detail model for iphone, ipod, ipad if the model information exist. Please check the [pr](https://github.com/amplitude/ua-parser-js/commit/adbac2be284234b0cf8ff22e17b2ecec2036a8b8).

Reply:
Reverted these - added back the capturing of iphone version and test cases

Reply:
Good catch, updated